providers/circleci/circleci.go

```diff
@@ -0,0 +1,70 @@
+package circleci
+
+import (
+	"fmt"
+	"net/url"
+	"path"
+
+	"github.com/KeisukeYamashita/biko/alias"
+	"github.com/urfave/cli"
+)
+
+// Provider ...
+type Provider struct {
+	baseURL *url.URL
+	URL     *url.URL
+	Ctx     *cli.Context
+	Aliases map[string]interface{}
+}
+
+// GetProvider ...
+func GetProvider() (*Provider, error) {
+	conf, err := alias.GetConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	return &Provider{
+		Aliases: conf.CircleCI["alias"].(map[string]interface{}),
+	}, nil
+}
+
+// Init ...
+func (p *Provider) Init(c *cli.Context) error {
+	p.Ctx = c
+	return nil
+}
+
+// GetTargetURL ...
+func (p *Provider) GetTargetURL() (string, error) {
+	var baseURL = fmt.Sprintf("https://circleci.com/gh/%s/", p.Ctx.String("org"))
```

Comment:
```suggestion
	var baseURL = fmt.Sprintf("https://circleci.com/gh/%s/", p.Org)
```

Reply:
If use `subcommand` , Can not set variable of `p.Org`. So I delete `p.Org` and use `p.Ctx.String("org")`. Do you have any idea?

Reply:
@shopetan Sorry for the PagerDuty example. You can do like this. 
https://github.com/KeisukeYamashita/biko/blob/dc2849c5ccd84ea410ffbf1af06ebcc8ddc33aa7/cli/pagerduty.go#L52-L59

As same to the `$ biko cc` command. Implement in the struct.